drivers/gpio/gpio_stm32.c

```diff
@@ -227,7 +244,7 @@ static void gpio_stm32_set_exti_source(int port, int pin)
 
 static int gpio_stm32_get_exti_source(int pin)
 {
-	uint32_t line = gpio_stm32_pin_to_exti_line(pin);
+	u32_t line = gpio_stm32_pin_to_exti_line(pin);
```

Comment:
code cleaning again?

Reply:
Indeed, not directly linked, but reported by CI.